src/index.js

```diff
@@ -4,7 +4,13 @@ import insertStyle, { removeStyle, getStyle } from './insert-style';
 
 const cache = new Map();
 
-export default function withStyles(css) {
+export default function (css, values) {
+
+  if (Array.isArray(css)) {
+    // Forward string literals to csjs
+    css = csjs(css, values);
```

Comment:
css is later passed to csjs.getStyles(). Normally the function wants a csjs object, but with this it simply converts string literals to that csjs object which it later needs.